client/js/components/statement/assessmentTable/DpExportModal.vue

```diff
@@ -62,6 +62,25 @@
                 v-model="exportChoice.pdf.anonymous" />
             </fieldset>
 
+            <fieldset
+              v-if="options.pdf.newPagePerStn && view === 'original_statements'"
+              class="u-mb-0_5 u-pb-0_5">
+<!--              <legend-->
+<!--                class="sr-only"-->
+<!--                v-text="" />-->
```

Comment:
Do we need this commented code here?

Reply:
Yes, the text was still missing. I added it here 67faa66eb3b555e13cbaefab60ab312adafbcdc8